### ISSUE TITLE
Loading without an end in sight.

### ISSUE THREAD

**Issue by m10ev** [ddPn08/rvc-webui#67]
# When changing any settings that require loading time or clicking the infer button, it loads without an end in sight.

## Image
![image](https://github.com/ddPn08/rvc-webui/assets/42208556/9328772a-ffa3-4f69-a606-24f60e61223d)

## Console output:
```
2023-07-13 00:30:05 | INFO | httpx | HTTP Request: POST http://ip:port/reset "HTTP/1.1 200 OK"
2023-07-13 00:37:50 | ERROR | asyncio | Task exception was never retrieved
future: <Task finished name='rjtxsra2mxc_0' coro=<Queue.process_events() done, defined at D:\rvc-webui-main\venv\lib\site-packages\gradio\queueing.py:343> exception=1 validation error for PredictBody
event_id
  Field required [type=missing, input_value={'fn_index': 0, 'data': [...on_hash': 'rjtxsra2mxc'}, input_type=dict]
    For further information visit https://errors.pydantic.dev/2.1.2/v/missing>
Traceback (most recent call last):
  File "D:\rvc-webui-main\venv\lib\site-packages\gradio\queueing.py", line 347, in process_events
    client_awake = await self.gather_event_data(event)
  File "D:\rvc-webui-main\venv\lib\site-packages\gradio\queueing.py", line 220, in gather_event_data
    data, client_awake = await self.get_message(event, timeout=receive_timeout)
  File "D:\rvc-webui-main\venv\lib\site-packages\gradio\queueing.py", line 456, in get_message
    return PredictBody(**data), True
  File "D:\rvc-webui-main\venv\lib\site-packages\pydantic\main.py", line 150, in __init__
    __pydantic_self__.__pydantic_validator__.validate_python(data, self_instance=__pydantic_self__)
pydantic_core._pydantic_core.ValidationError: 1 validation error for PredictBody
event_id
  Field required [type=missing, input_value={'fn_index': 0, 'data': [...on_hash': 'rjtxsra2mxc'}, input_type=dict]
    For further information visit https://errors.pydantic.dev/2.1.2/v/missing
2023-07-13 00:37:50 | INFO | httpx | HTTP Request: POST http://ip:port/reset "HTTP/1.1 200 OK"
2023-07-13 00:37:55 | INFO | httpx | HTTP Request: POST http://ip:port/reset "HTTP/1.1 200 OK"
2023-07-13 00:42:07 | ERROR | asyncio | Task exception was never retrieved
future: <Task finished name='rjtxsra2mxc_2' coro=<Queue.process_events() done, defined at D:\rvc-webui-main\venv\lib\site-packages\gradio\queueing.py:343> exception=1 validation error for PredictBody
event_id
  Field required [type=missing, input_value={'fn_index': 2, 'data': [...on_hash': 'rjtxsra2mxc'}, input_type=dict]
    For further information visit https://errors.pydantic.dev/2.1.2/v/missing>
Traceback (most recent call last):
  File "D:\rvc-webui-main\venv\lib\site-packages\gradio\queueing.py", line 347, in process_events
    client_awake = await self.gather_event_data(event)
  File "D:\rvc-webui-main\venv\lib\site-packages\gradio\queueing.py", line 220, in gather_event_data
    data, client_awake = await self.get_message(event, timeout=receive_timeout)
  File "D:\rvc-webui-main\venv\lib\site-packages\gradio\queueing.py", line 456, in get_message
    return PredictBody(**data), True
  File "D:\rvc-webui-main\venv\lib\site-packages\pydantic\main.py", line 150, in __init__
    __pydantic_self__.__pydantic_validator__.validate_python(data, self_instance=__pydantic_self__)
pydantic_core._pydantic_core.ValidationError: 1 validation error for PredictBody
event_id
  Field required [type=missing, input_value={'fn_index': 2, 'data': [...on_hash': 'rjtxsra2mxc'}, input_type=dict]
    For further information visit https://errors.pydantic.dev/2.1.2/v/missing
```

**Comment by ddPn08** [comment]
Fixed in 3aed3783879637ad4d73ddb7b393857d674c2adc .

**Comment by m10ev** [comment]
Confirmation: The issue has been resolved.

Thank you!

Closing Issue.
